src/main/java/duke/command/CalendarCommand.java

```diff
@@ -0,0 +1,65 @@
+package duke.command;
+
+import duke.exception.UnknownDateTimeException;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CalendarCommand implements Command {
+    /**
+     * calendarCommand Method finds Tasks that finds, matches and displays Tasks
+     * on a specific date.
+     *
+     * @param taskList      is the list of Tasks are saved and manipulated
+     * @param commandSuffix is the additional String that accompanies two-step commands
+     * @throws UnknownDateTimeException when improperly formatted DateTime values are given
+     */
+    public static String run(TaskList taskList, String commandSuffix) throws UnknownDateTimeException {
+        try {
+            DateTimeFormatter dtFormat = DateTimeFormatter.ofPattern("d/M/yyyy");
```

Comment:
Personally, I will make it `dateTimeFormat` instead of `dtFormat`, because `dtFormat` can actually is a variable for a real `DT Format` object. Preference aside, perhaps this example can highlight my point: would you name UTC time format `utc` or `Utc` or `UTC`? Haha hope you get my point.

Reply:
The formatting was forced this way due to the standardised `checkstyle.xml` file not agreeing with variables having more than 3 consecutive capital letters. With regards to the `dtFormat` being an actual variable for the real "`DT Format` object" - that is intended, as you can see used in the code (i.e. where your UTC analogy can be presented by `utc` or `UTC` by convention, with only `Utc` breaking that premise, because variables don't start with a capital letter in the current checkstyle).